README.md

```diff
@@ -607,7 +623,19 @@ _Examples_:
 
 Binary files can be specified in the configuration map in their base64 encoded format with a basename having a `.base64` extension. Such files are then converted back to binary as they are copied to their target path.
 
-When an /etc/sssd/sssd.conf file is included in the configuration map, the httpd pod automatically enables the sssd service upon startup.
```

Comment:
Is this no longer true? Why remove this line? I feel like the table doesn't make this point clear.